### PR TITLE
feat(posthog) Add post hog inserts

### PR DESCRIPTION
**Pull request by devcodedcom**

### CODE REVIEW

File: themes/my-theme/scripts/image.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ hexo.extend.tag.register('banner_ad', function (args) {
   var assetPath = args[0];
   var targetUrl = args[1];
 
-  return `<a class="course-image" href="${targetUrl}" rel="nofollow" aria-label="Dev Academy courses">
+  return `<a class="course-image" href="${targetUrl}" rel="nofollow" data-ph="banner-ad__link" aria-label="Dev Academy courses">
```
Comment (bartosz-io) [inline themes/my-theme/scripts/image.js:67]: `banner-ad` name is not a good idea -> might be removed by ad blockers!

```suggestion
  return `<a class="course-image" href="${targetUrl}" rel="nofollow" data-ph="banner-course__link" aria-label="Dev Academy courses">
```

File: themes/my-theme/source/js/posthog/selectors.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- subscribe-for-more__link
+- show-comments__button
+- article-sidebar-contributor__link_fullname
+- article-sidebar-contributor-academies__link_wsa
+- article-sidebar-contributor-academies__link_fta
+
+Optional:
+
+- review-screen__link
+- banner-ad__link
```
Comment (bartosz-io) [inline themes/my-theme/source/js/posthog/selectors.md:97]:
```suggestion
- banner-course__link

```

File: README.md [hunk 3]
```diff
@@ -45,4 +45,19 @@ contributors:
...
+- data-ph='{block}__{element}\_{modifier}_{value}'
+
+Czyli na przykład:
+- data-ph='header__logo'
+- data-ph='nav__link_courses'
+- data-ph='nav__link_academies_wsa'
+- data-ph='footer-nav__link_contact_linkedin'
+
+### Blokowanie przechwytywania danych
+- Dodanie klasy *ph-no-capture* zablokuje danych element.
```
Comment (bartosz-io) [inline README.md:62]: This should be added to email form fields, right?

**Review by bartosz-io — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (devcodedcom) [reply README.md:62]: Every anchor, label, button, inputs, textareas etc.

**Review by bartosz-io — APPROVED** [review]

**Pull request merged**